com.unity.shadergraph/Editor/Drawing/Interfaces/IResizable.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace UnityEditor.ShaderGraph.Drawing.Interfaces
+{
+    interface IResizable
```

Comment:
I don't really like just using the same name - I know it's in a different namespace but seems unnecessarily confusing. Maybe "ISGResizable" or something? And you can inherit the original interface so it's clear it's an extension and then our ISGResizable classes can also be passed to GV Resizing functions (we may not do that but it's cleaner).

Ah, OK, gross - the GV one was added by VFX team, probably after we made ours here. Well, since we're pulling ours out to its own class, I think still best to inherit from the GV version just to avoid confusing duplication. =)

Reply:
Oh that's a good idea, honestly didn't even occur to me to inherit from the GV version, will do that and rename ours to `ISGResizable` to avoid naming ambiguity.